lib/std/fs.zig

```diff
@@ -28,6 +28,8 @@ pub const realpathW = os.realpathW;
 pub const getAppDataDir = @import("fs/get_app_data_dir.zig").getAppDataDir;
 pub const GetAppDataDirError = @import("fs/get_app_data_dir.zig").GetAppDataDirError;
 
+pub const DeleteFileError = os.UnlinkError;
```

Comment:
All the other functions in the file (eg. `createFile`) refer to the qualified error set name, prepending `Dir.` to `DeleteFileError` is enough.

Adding a test case is also greatly appreciated!

Reply:
Given that functions like `createFile` use `File.OpenError`, maybe the DeleteFileError should be moved into `File` as `File.DeleteError` instead?

Reply:
I'm gonna push a version that does what you suggested and adds a test, I feel like moving the DeleteFileError would be a case for an another PR.